README.md

```diff
@@ -30,8 +37,193 @@ Agrega la siguiente dependencia en el archivo pom de tu proyecto Maven:
+sitio, esta vez a la URL que indicaste en el `finalUrl` cuando iniciaste la
+transacción. Tal como antes, recibirás el `ws_token` que te permitirá
+identificar la transacción y mostrar un comprobante o página de éxito a tu
+usuario.
+
+#### Otras funcionalidades de Webpay Plus
+
+Eso concluye lo mínimo para crear y confirmar una transacción Webpay Plus.
+En [doc/WebpayPlus.md](doc/WebpayPlus.md) puedes encontrar más información sobre otras funcionalidades disponibles para Webpay Plus.
+
+### Webpay OneClick
+
+#### Crear una transacción Webpay OneClick
+
+Para usar Webpay Onelick en transacciones asociadas a a un único comercio, lo
+primero que necesitas es preparar una instancia de `WebpayOneClick` con la
+`Configuration` que incluye el código de comercio y los certificados a
+usar
+
+Una forma fácil de comenzar es usar la configuración para pruebas que viene
+incluida en el SDK:
+
+```java
+import cl.transbank.webpay.configuration.Configuration;
+import cl.transbank.webpay.Webpay;
+import cl.transbank.webpay.WebpayOneClick;
+// ...
+
+WebpayOneClick transaction =
+    new Webpay(Configuration.forTestingWebpayOneClickNormal()).getOneClickTransaction();
+```
+
+> **Tip**: Como necesitarás ese objeto `transaction` en múltiples ocasiones, es buena idea
+encapsular la lógica que lo genera en algún método que puedas reutilizar.
+
+Una vez que ya cuentas con esa preparación, puedes iniciar transacciones:
+
+```java
+import com.transbank.webpayserver.webservices.OneClickInscriptionOutput;
+//...
+
+String username = "identificador-del-usuario-en-comercio";
+String email = "email@del.usuario";
+String urlReturn = "https://callback/resultado/de/transaccion";
+OneClickInscriptionOutput initResult =
+    transaction.initInscription(username, email, urlReturn);
+String formAction = initResult.getUrl();
+String tbkToken = initResult.getToken();
+```
+
+Tal como en el caso de Webpay Plus, debes redireccionar via `POST` el
+navegador del usuario a la url retornada en `initInscription`. A diferencia
+de Webpay Plus, acá el nombre del parámetro que contiene el token se debe
+llamar `TBK_TOKEN`.
+
+Una vez que el usuario autorice la inscripción, retornará el control al
+comercio via `POST` en la url indicada en `urlReturn`, con el parámetro
+`TBK_TOKEN` identificando la transacción. Con esa información se puede
+finalizar la inscripción:
+
+```java
+import com.transbank.webpayserver.webservices.OneClickFinishInscriptionOutput;
+//...
+
+OneClickFinishInscriptionOutput result =
+    transaction.finishInscription(tbkToken);
+if (result.responseCode == 0) {
+    // Inscripcion exitosa.
+    // Ahora puedes usar result.tbkUser para autorizar transacciones
+    // oneclick sin nueva intervención del usuario.
+}
+```
+
+Finalmente, puedes autorizar transacciones usando el `tbkUser` retornado:
+
+```java
+import com.transbank.webpayserver.webservices.OneClickPayOutput;
+//...
+Long buyOrder = 1234; // identificador único de orden de compra;
```

Comment:
Is buyOrder auto generated or it's the commerce who must to generate it? Is it unique for commerce or it is unique for webpay?

Reply:
The merchant ("commerce" is a bad translation for comercio really, but unfortunately there are a lot of code and parameters that use that bad translation) must generate it. I'll make that clearer.

It's better if it's unique for the merchant (it's supposed to be the real "orden de compra" number)  but webpay can't enforce that. So I'd prefer to not write that complicated stuff in the README and just say it is unique. If the merchant decides to keep it unique for webpay it will still work.